multiqc/modules/fastqc/fastqc.py

```diff
@@ -317,7 +317,7 @@ def per_seq_quality_plot (self):
         pconfig = {
             'id': 'fastqc_per_sequence_quality_scores_plot',
             'title': 'FastQC: Per Sequence Quality Scores',
-            'ylab': 'Count',
+            'ylab': 'Percentage',
```

Comment:
Is this percentages? In the plot I'm looking at it goes up to 300 million so I'm pretty sure it's counts for me. What does the FastQC report say?